pkgs/development/web/nodejs/nodejs.nix

```diff
@@ -39,6 +39,10 @@ let
       (name: "${lib.getDev sharedLibDeps.${name}}/include/*")
       (builtins.attrNames sharedLibDeps);
 
+  buildPkgs = if stdenv.buildPlatform.isx86_64 && stdenv.is32bit
+    then buildPackages.pkgsi686Linux.buildPackages
+    else buildPackages;
```

Comment:
(Small note, that will be irrelevant if we use the callPackage version instead.) This is incorrect.  buildPackages is equivalent to nativeBuildInputs.  For depsBuildBuild, packages are sourced from pkgsBuildBuild.

Reply:
Call package version used instead.